OpenRA.Mods.Common/Activities/Air/TakeOff.cs

```diff
@@ -97,6 +97,9 @@ public override Activity Tick(Actor self)
 			// Checking for NextActivity == null again in case another activity was queued while taking off
 			if (moveToRallyPoint && NextActivity == null)
 			{
+				if (!aircraft.Info.VTOL && assignTargetOnFirstRun)
+					return new FlyTimed(1, self);
```

Comment:
The idea was to nudge the aircraft into the air so the idle behaviour could take over. It shouldn't perform the `AttackMoveActivity` below because target defaults to `Self.Location` and that doesn't really make sense for a non-VTOL aircraft. I think the recent changes to `TakeOff` have made this partially redundant, because the block above seems to already do the undirected takeoff. However, this still needs a conditional to skip the `AttackMoveActivity`

Reply:
I'm thinking mainly about compatibility with e98dba2d0a21b16d508c0f2b5fb5e5e3a72d537b

Reply:
Yes, it was a mistake in that respect. I should have used childactivities for this. But like I said, it has probably been made redundant anyway.

Reply:
Fixed now.